1.1/spec/11-Part-08.adoc

```diff
@@ -103,12 +103,60 @@ WHERE { :f1 geo:sfDisjoint :f1 }
 
 Such cases are application-specific data modeling errors and are therefore outside of the scope of the GeoSPARQL specification.
 
-====  Standard Properties for geo:Geometry
+===== Property: geo:hasLength
+
+The property `geo:hasLength` is used to indicate the length of a `geo:Feature`. The property indicates `geo:SpatialMeasure` object which then contains the length value expressed as a scalar number and which potentially also includes units of measure, uncertainty and other properties.
```

Comment:
2nd sentence: 

The range of the property is `geo:SpatialMeasure`, which encodes the length value expressed as a scalar quantity and which also includes the units of measure, and potentially uncertainty and other properties.

Reply:
Fixed with https://github.com/opengeospatial/ogc-geosparql/pull/72/commits/b79b1d5da4562d0c9dc13aa914d440eb60618534